Index: blog/admin.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from django.contrib import admin\n\n# Register your models here.\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/blog/admin.py b/blog/admin.py
--- a/blog/admin.py	(revision 5d991e00da5a0be53cb44cb58bff1aad03c65059)
+++ b/blog/admin.py	(date 1672357387318)
@@ -1,3 +1,48 @@
 from django.contrib import admin
+from .models import Post, Comments
+from django.contrib.auth.models import User
+
 
-# Register your models here.
+class CommentsInlineModelAdmin(admin.TabularInline):
+    model = Comments
+    fk_name = 'post'
+
+
+# class PostInlineModelAdmin(admin.TabularInline):
+#     model = Post
+#
+#
+# class UserInlineModelAdmin(admin.TabularInline):
+#     model = User
+
+
+class CommentsAdmin(admin.ModelAdmin):
+    list_display = ['username', 'text', 'published']
+    list_filter = ['published']
+    list_per_page = 20
+    fields = ['username', 'post', 'text', 'published']
+    search_fields = ['username']
+    # inlines = [PostInlineModelAdmin]
+    save_as = True
+
+
+class PostAdmin(admin.ModelAdmin):
+    list_display = ['user', 'title', 'text', 'published']
+    list_filter = ['published']
+    search_fields = ['user']
+    date_hierarchy = 'created_date'
+    list_per_page = 20
+    fieldsets = (
+        ('Post info', {
+            'fields': ('title', 'text', 'created_date', 'published')
+        }),
+        ('User', {
+            'fields': ('user',)
+         }),
+    )
+    inlines = [CommentsInlineModelAdmin]
+    save_as = True
+
+
+admin.site.register(Post, PostAdmin)
+admin.site.register(Comments, CommentsAdmin)
